rustworkx-core/src/shortest_path/dijkstra.rs

```diff
@@ -102,7 +102,7 @@ pub fn dijkstra<G, F, K, E, S>(
     start: G::NodeId,
     goal: Option<G::NodeId>,
     mut edge_cost: F,
-    mut path: Option<&mut DictMap<G::NodeId, Vec<G::NodeId>>>,
+    mut path: Option<&mut DictMap<G::NodeId, Vec<Vec<G::NodeId>>>>,
```

Comment:
You should create a new function instead of altering the existing one. This would be a breaking change